### PR TITLE
Fix ios chrome ime double input issue.

### PR DESCRIPTION
**Pull request by ulion**
#### Is this adding or improving a _feature_ or fixing a _bug_?

Fix a bug

#### What's the new behavior?

IME input won't duplicate.

#### How does this change work?

IOS chrome looks just fine, so no need special code on compositionEnd to insertData.

#### Have you checked that...?

- [x] The new code matches the existing patterns and styles.
- [x] The tests pass with `yarn test`.
- [x] The linter passes with `yarn lint`. (Fix errors with `yarn fix`.)
- [x] The relevant examples still work. (Run examples with `yarn start`.)

#### Does this fix any issues or need any specific reviewers?

Fixes: #
Reviewers: @

### CODE REVIEW

**Comment by ianstormtaylor** [comment]
Thanks @ulion!

File: packages/slate-react/src/components/editable.tsx [hunk 1]
```diff
@@ -613,7 +614,7 @@ export const Editable = (props: EditableProps) => {
               // aren't correct and never fire the "insertFromComposition"
               // type that we need. So instead, insert whenever a composition
               // ends since it will already have been committed to the DOM.
-              if (!IS_SAFARI && !IS_FIREFOX && event.data) {
+              if (!IS_SAFARI && !IS_FIREFOX && !IS_IOS && event.data) {
```
Comment (dylans) [inline packages/slate-react/src/components/editable.tsx:617]: This looks fine, but it's stale as we recently updated it to IS_FIREFOX_LEGACY. Could you do a quick rebase and then we can land this? Thanks @ulion!

**Review by dylans — CHANGES_REQUESTED** [review]

**Comment by ulion** [comment]
rebased

Dylan Schiemann ***@***.***> 于2021年8月4日周三 下午9:00写道：

**Review by dylans — APPROVED** [review]

**Comment by dylans** [comment]
Thanks @ulion, I'm making a push to land obvious PRs asap, so I hope to get this merged very soon.

**Comment by dylans** [comment]
@ulion  This one was missing a changeset so I've added a PR on your branch. If you don't mind please accept and push into this PR and then I'll land it.

File: .changeset/metal-trees-rest.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+---
+"slate-react": patch
+"slate": patch
+---
+
+Fix ios chrome ime double input issue.
```
Comment (dylans) [inline .changeset/metal-trees-rest.md:6]:
```suggestion
fix ios chrome ime double input issue
```

Comment (dylans) [reply .changeset/metal-trees-rest.md:6]: My fault, I failed to run prettier on the changeset... sigh.

File: .changeset/metal-trees-rest.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+---
+"slate-react": patch
```
Comment (dylans) [inline .changeset/metal-trees-rest.md:2]:
```suggestion
'slate-react': patch
```

File: .changeset/metal-trees-rest.md [hunk 4]
```diff
@@ -0,0 +1,6 @@
...
+"slate": patch
```
Comment (dylans) [inline .changeset/metal-trees-rest.md:3]:
```suggestion
'slate': patch
```

**Comment by dylans** [comment]
@ulion sorry about the prettier issue, if you can please run `yarn lint:prettier --write` on the PR and push then I can merge. Or just give me edit permission on the PR and I'll fix and merge. Thanks.

**Comment by ulion** [comment]
> @ulion sorry about the prettier issue, if you can please run `yarn lint:prettier --write` on the PR and push then I can merge. Or just give me edit permission on the PR and I'll fix and merge. Thanks.

done both.

**Pull request merged**